### PR TITLE
Feature/holiday approval

### PR DESCRIPTION
**Pull request by William-Unosquare**
image

- Adds sidebar and routing to sub-pages within the admin panel, linking to different features. 
- Adds admin feature to approve and reject holidays. Right now it's just the admin role users that can do this, but we can serve a slightly different version of the same admin page depending on if you're a team leader in the future.  Again, this is just going to be good to demo the app even if my approach is completely wrong, so I thought it would be worth getting in.

### CODE REVIEW

File: web-app/src/components/common/Errorbox/styled.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
 import styled from 'styled-components';
 
 export const ErrorContainer = styled.div`
+<<<<<<< HEAD
```
Comment (bgribz) [inline web-app/src/components/common/Errorbox/styled.js]: :(

**Review by bgribz — CHANGES_REQUESTED** [review]

File: web-app/src/components/common/Errorbox/styled.js (hunk 1 above)
Comment (William-Unosquare) [reply web-app/src/components/common/Errorbox/styled.js]: This component's merge conflict will be the end of me.

File: web-app/src/components/PendingHolidays/container.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      return holidays.filter(
+        holiday => holiday.holidayStatusId === holidayStatus.PENDING,
+      );
+    };
+
+    approveHoliday = holiday => {
+      const holidayIndex = this.state.pendingHolidays.indexOf(holiday);
+      const pendingHolidays = [...this.state.pendingHolidays];
+      const approvedHoliday = { ...holiday };
+      approvedHoliday.holidayStatusId = holidayStatus.APPROVED;
```
Comment (BrownMark) [inline web-app/src/components/PendingHolidays/container.js]: this should probably be within the promise resolve.

File: web-app/src/components/PendingHolidays/container.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            type: 'error',
+          }),
+        );
+    };
+
+    rejectHoliday = holiday => {
+      const holidayIndex = this.state.pendingHolidays.indexOf(holiday);
+      const pendingHolidays = [...this.state.pendingHolidays];
+      const rejectedHoliday = { ...holiday };
+      rejectedHoliday.holidayStatusId = holidayStatus.REJECTED;
```
Comment (BrownMark) [inline web-app/src/components/PendingHolidays/container.js]: again i would put this in the resolve.

**Review by BrownMark — CHANGES_REQUESTED** [review]

**Review by BrownMark — APPROVED** [review]

**Review by bgribz — APPROVED** [review]

**Pull request merged**